tempesta_fw/t/functional/helpers/control.py

```diff
@@ -123,7 +123,7 @@ def append_script_option(self):
                         '/../wrk/', self.script, '.lua'])
         script_path = os.path.abspath(path)
         assert os.path.isfile(script_path), \
-               'No script found: %s !' % script_path
+            'No script found: %s !' % script_path
```

Comment:
This is an unrelated change. Please revert this hunk.

Reply:
Corrected.